t/api/02-iso.t

```diff
@@ -185,6 +185,26 @@ is($ret->tx->res->json->{job}->{state}, 'cancelled', "job $newid is cancelled");
 # make sure we can't post invalid parameters
 $ret = $t->post_ok('/api/v1/isos', form => {iso => $iso, tests => "kde/usb"})->status_is(400);
 
+# handle list of tests
+my $iso2 = 'openSUSE-13.1-DVD-i586-Build0093-Media.iso';
+$ret = $t->post_ok(
+    '/api/v1/isos',
+    form => {
+        ISO     => $iso2,
+        DISTRI  => 'opensuse',
+        VERSION => '13.1',
+        FLAVOR  => 'DVD',
+        ARCH    => 'i586',
+        TEST    => 'server,kde,textmode',
+        BUILD   => '0093'
+    })->status_is(200);
+
+$ret = $t->get_ok('/api/v1/jobs');
+is($ret->tx->res->json->{count}, 3, "3 new jobs created");
```

Comment:
from where did you get `count` field? `/api/v1/jobs` returns json with `jobs` field as a list of job hashes. So you want something like
```perl
scalar @{$ret->tx->res->json->{jobs}}
```